src/pages/dogs/DogDetailsPage.tsx

```diff
@@ -36,6 +36,7 @@ interface DogDetailsReturnType {
   color: string;
   size: string;
   sex: "male" | "female";
+  ageGroup: "puppy" | "grown" | "old";
```

Comment:
Can you rename this to "puppy", "adult" and "senior"? As these are the conventional terms